core/templates/dev/head/pages/exploration-editor-page/modal-templates/post-publish-modal.template.html

```diff
@@ -66,8 +66,10 @@ <h3>Awesome!</h3>
       Now that you're a published Oppia teacher, you can share your creation.
     </p>
     <div class="oppia-share-publish-link">
-      Here's your exploration link:
+      Here's your exploration link:<br>
+      <span style="visibility:hidden" ng-if="copied==true">&nbsp Copied &nbsp<i class="fa fa-check" aria-hidden="true"></i></span>
       <div ng-click="selectText($event)"><[explorationLink]></div>
+      <span ng-if="copied==true">&nbsp Copied &nbsp<i class="fa fa-check" aria-hidden="true"></i></span>
```

Comment:
I'm not sure what's the line 70 and 72 are supposed to do? Can you please explain?

Reply:
Line 72 is just the confirmation message when the use user clicks on the link and it displays copied.
If I only add the line 72, the exploration link is displaced horizontally:

![](https://user-images.githubusercontent.com/45149542/61369373-9559a700-a8ad-11e9-94c8-d24068ba6eb5.png)

So line 70 is used only to balance the text on the left side, such that the exploration link is not displaced. It will be hidden and has the same text as the link 72. After balancing the final webpage looks like this:

![](https://user-images.githubusercontent.com/45149542/61551532-11511c00-aa73-11e9-8060-b1d438e9000d.png)